pkgs/applications/science/machine-learning/vowpal-wabbit/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, stdenv, fetchgit, cmake, boost169, zlib }:
+
+let
+  version = "8.9.2";
+in
+stdenv.mkDerivation {
+  pname = "vowpal-wabbit";
+  version = "${version}";
+  src =
+    fetchgit {
```

Comment:
Please use `fetchFromGitHub` instead. This could also be on the same line.

Reply:
I wasn't able to use `fetchFromGitHub` as it seems like it does not support downloading submodules

Reply:
It does, you can use `fetchSubmodules` as well.

Reply:
Ah I see. Let me try it again then